Extension/package.json

```diff
@@ -2761,6 +2761,17 @@
           "default": false,
           "markdownDescription": "%c_cpp.configuration.legacyCompilerArgsBehavior.markdownDescription%",
           "scope": "resource"
+        },
+        "C_Cpp.sshTargetsView": {
+          "type": "string",
+          "enum": [
+            "enabled",
+            "disabled",
+            "auto"
```

Comment:
I think "auto" should be "default" to be consistent with the other setting enums. Or is there are reason for choosing "auto" over "default"?

Reply:
Personally I prefer "auto", since "default" is less descriptive and requires more explanation, but if "default" is a convention, I'm ok with it.